### ISSUE TITLE
Wizard Behaviour - wizard not talking to extent selector

### ISSUE THREAD

**Issue by Charlotte-Morgan** [AIFDR/inasafe#2311]
I am testing the Impact Function Centric Wizard and found 2 problems

---------------------------------------------------------------------
# problem 1 - minor
scenario: QGIS has hazard  data loaded without  kw and no exposure data loaded
The path through wizard is to use hazard data loaded in QGIS but step out to assign KW
- this works fine except once KW are assigned - usre is asked the same question again, ie identify your hazard data (nothing acknowledges the path through the KW wizard and success on assigning kw) - works but could be confusing
User is then asked to find exposure data on disk and again steps through the kw wizard.
- this works fine except once KW are assigned - user is asked the same question again, ie identify your exposure data from disk (nothing acknowledges the path through the KW wizard and success on assigning kw) - works but could be confusing. As a user I would assume that choosing the data the first time would load it ...

# problem 2 - major
... set analysis extent - wizard steps out to extent selector. However extent selector now offers many options, more that just select area on map.
If i choose - define the extent on the map 
![image](https://cloud.githubusercontent.com/assets/5743135/9648743/1a3dc7be-5216-11e5-8b0e-8416b35a18a3.png)

& then in the set analysis area option choose use the intersection of hazard and exposure
![image](https://cloud.githubusercontent.com/assets/5743135/9648760/3a11c072-5216-11e5-94c9-5e2f1a6ed55f.png)

i get this

```
An error has occurred while executing Python code:


Traceback (most recent call last):
  File "C:/Users/AIFDR_ADMIN/.qgis2/python/plugins\inasafe\safe\gui\tools\wizard_dialog.py", line 3599, in on_pbnNext_released
    new_step = self.compute_next_step(current_step)
  File "C:/Users/AIFDR_ADMIN/.qgis2/python/plugins\inasafe\safe\gui\tools\wizard_dialog.py", line 3950, in compute_next_step
    if self.validate_extent():
  File "C:/Users/AIFDR_ADMIN/.qgis2/python/plugins\inasafe\safe\gui\tools\wizard_dialog.py", line 3301, in validate_extent
    return extent.intersects(haz_extent) and extent.intersects(exp_extent)
AttributeError: 'NoneType' object has no attribute 'intersects'


Python version:
2.7.5 (default, May 15 2013, 22:44:16) [MSC v.1500 64 bit (AMD64)]


QGIS version:
2.8.3-Wien Wien, 313a6bc

Python path: ['C:/Users/AIFDR_ADMIN/.qgis2/python/plugins\\processing', 'C:/PROGRA~1/QGISWI~1/apps/qgis-ltr/./python', u'C:/Users/AIFDR_ADMIN/.qgis2/python', u'C:/Users/AIFDR_ADMIN/.qgis2/python/plugins', 'C:/PROGRA~1/QGISWI~1/apps/qgis-ltr/./python/plugins', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\matplotlib-1.3.1-py2.7-win-amd64.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\nose-1.3.3-py2.7.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\tornado-4.0.1-py2.7-win-amd64.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\backports.ssl_match_hostname-3.4.0.2-py2.7.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\certifi-14.05.14-py2.7.egg', 'C:\\Program Files (x86)\\ArcGIS\\Desktop10.2\\bin', 'C:\\Program Files (x86)\\ArcGIS\\Desktop10.2\\arcpy', 'C:\\Program Files (x86)\\ArcGIS\\Desktop10.2\\ArcToolbox', 'C:\\PROGRA~1\\QGISWI~1\\bin\\python27.zip', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\DLLs', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\plat-win', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\lib-tk', 'C:\\PROGRA~1\\QGISWI~1\\bin', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\PIL', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\jinja2-2.7.2-py2.7.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\markupsafe-0.23-py2.7-win-amd64.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\pytz-2012j-py2.7.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\win32', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\win32\\lib', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\Pythonwin', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\Shapely-1.2.18-py2.7-win-amd64.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\six-1.3.0-py2.7.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\wx-2.8-msw-unicode', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\xlrd-0.9.2-py2.7.egg', 'C:\\PROGRA~1\\QGISWI~1\\apps\\Python27\\lib\\site-packages\\xlwt-0.7.5-py2.7.egg', u'C:/Users/AIFDR_ADMIN/.qgis2//python', 'C:/Users/AIFDR_ADMIN/.qgis2/python/plugins\\inasafe', 'C:\\Users\\AIFDR_ADMIN\\.qgis2\\python\\plugins\\inasafe\\safe_extras\\parameters', 'C:\\Users\\AIFDR_ADMIN\\.qgis2\\python\\plugins', 'C:\\Users\\AIFDR_ADMIN\\.qgis2\\python\\plugins\\inasafe\\safe_extras', 'C:\\PROGRA~1\\QGISWI~1\\apps\\qgis-ltr\\python\\plugins\\fTools\\tools']
```
However: if i select use extent of map layers in the wizard, it progresses without an error
Also - if i use the extent selector to drag on the map, it also works

# Proposed solution - problem 2
- [ ] Wizard can read all options in the extent selector not just the drag on map
- [ ] wizard must step out to extent selector and not offer other options 

# proposed solution - problem 1
try to not confuse users - better wording in BFR 321 (bug fix release 3.2.1)

# cc
@timlinux @ismailsunni

**Comment by Charlotte-Morgan** [comment]
the wizard pane that steps out to the extent selector should have the same text in the string as in the linked text
eg change "define it now" to "define the analysis extent now"

**Comment by ismailsunni** [comment]
Seems similar with #2261

**Comment by borysiasty** [comment]
Problem 2 is fixed in PR #2346. I'm moving the minor one to a separate ticket.
